lib/packages.cf

```diff
@@ -1494,7 +1494,7 @@ body package_method freebsd
       package_changes => "individual";
 
       # Could use rpm for this
-      package_list_command => "/usr/sbin/pkg_info";
+      package_list_command => "/usr/sbin/pkg info";
```

Comment:
On Tue, Feb 23, 2016 at 01:24:49PM -0800, Nick Anderson wrote:
> When did these commands change? Do we need to account for backward
> compatibility for the older clients that only have `pkg_info`?
> 

pkg_info has been deprecated since Oct. 2013 
(https://lists.freebsd.org/pipermail/freebsd-pkg/2013-October/000107.html).

End of life for pkg_* was announced in Feb. 2014 
(https://lists.freebsd.org/pipermail/freebsd-ports-announce/2014-February/000077.html).

While /usr/sbin/pkg_info is still present in the 9.x-RELEASE series,
it is not present in 10.0-RELEASE and later.  The oldest supported
FreeBSD release, at this writing, is 9.3 which was released in July of
2014.  The expected End-of-Life for FreeBSD 9-stable is 31 Dec. 2016.

I believe that all FreeBSD systems running 9.0-RELEASE or later can be
safely assumed to have /usr/sbin/pkg available.  

-- 
John D. "Trix" Farrar            __\\|//__               Basement.NET
trix@basement.net                (` o-o ')   http://www.basement.net/
--------------------------------ooO-(_)-Ooo--------------------------
 GPG Key Fprint: 525F DBA7 1A62 E4C4 E642  DF95 384B B851 3CEF C10A